src/components/organisms/Room/Room.tsx

```diff
@@ -60,16 +59,12 @@ export const Room: React.FC<RoomProps> = ({
       userWithId && (
         <div
           className={`participant-container ${participantContainerClassName}`}
-          key={localParticipant.identity}
+          key={localParticipant.sparkleId}
         >
-          <VideoParticipant
-            participant={localParticipant}
-            participantUser={userWithId}
-            defaultMute={defaultMute}
-          />
+          <VideoCommsParticipant participant={localParticipant} isLocal />
```

Comment:
`isLocal` might make sense seeing it as a local variable inside a component, but like this it just stands out. Plenty of other props are just named `disabled`, `autoPlay` etc.

Could it be something like `location="local"` or `location="remote"` or maybe just shortened to `local` or `remote`?